bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -75,11 +75,11 @@
 	</channel-type>
 
 	<channel-type id="fanspeed" advanced="true">
-		<item-type>Number:Frequency</item-type>
```

Comment:
Why do you remove `Number:Frequency` ?

Reply:
I will restore it and then use unitHint feature.

Reply:
Changed